examples/js/loaders/3MFLoader.js

```diff
@@ -1,3 +1,7 @@
+/**
+ * @author technohippy / https://github.com/technohippy
+ */
+
 THREE.ThreeMFLoader = function ( manager ) {
```

Comment:
Would it make more sense if we use this pattern for loaders that start with a number?

```js
new THREE['3DSLoader']()
```